src/actions/searchAction.js

```diff
@@ -0,0 +1,18 @@
+// export default addUsername = e => ({ 
+//     type: "ADD_A_USERNAME",
+//     payload: { username: e.target.value }
+// })
+
+
+export async function fetchUserProfile(inputValue) {
```

Comment:
the async await function is not really working which is causing the github api to not connect. 
function fetchUserProfile(inputValue) {
    const url = `https:api.github.com/users/${inputValue}/repos`
    fetch(url)
        .then(res => res.json())
        .then(data => {
            console.log(data)
        })
}
export default fetchUserProfile

Reply:
function fetchUserProfile(inputValue) {
    const url = `https:api.github.com/users/${inputValue}/repos`
    fetch(url)
        .then(res => res.json())
        .then(data => {
            console.log(data)
        })
}
export default fetchUserProfile

Reply:
also put back:    return data[0].name